war/src/main/webapp/scripts/hudson-behavior.js

```diff
@@ -2338,6 +2346,15 @@ var hoverNotification = (function() {
     };
 })();
 
+// Decrease vertical padding for checkboxes
+window.addEventListener('load', function () {
+    document.querySelectorAll(".jenkins-form-item").forEach(function (element) {
+        if (element.querySelector(":scope > .optionalBlock-container > .row-group-start input[type='checkbox'], :scope > .optional-block-start input[type='checkbox'], :scope > div > .jenkins-checkbox") != null) {
```

Comment:
HtmlUnit can't handle this syntax unfortunately

See `hudson.PluginManagerTest#uploadDependencyResolution`